contracts/src/FeeContract.sol

```diff
@@ -49,8 +48,6 @@ contract FeeContract is Initializable, OwnableUpgradeable, UUPSUpgradeable {
     function initialize(address multisig) public initializer {
         __Ownable_init(multisig); //sets owner to msg.sender
         __UUPSUpgradeable_init();
-        maxDepositAmount = 1 ether;
```

Comment:
see comment above but these values have to set here as opposed to assigning it upon declaration of the state variable.